backend/WebApp/RenderWebApp.ts

```diff
@@ -41,7 +41,15 @@ export const renderWebApp = async (req, res: Response) => {
 
   stream.on("error", (err) => {
     console.log("ERROR", err);
+    res.end();
```

Comment:
This likely will need to be improved in the future - we should send an error page rather than just ending the http response.